docs/codeql/codeql-for-visual-studio-code/running-codeql-queries-at-scale-with-mrva.rst

```diff
@@ -153,6 +153,32 @@ For example, if you want to continue analyzing a set of repositories that had re
 
 You can then insert the ``new-repo-list`` of repositories into your list of custom repository lists for easy access in the Variant Analysis Repositories panel.
 
+Using code search to add repositories to a custom list 
```

Comment:
I think it might be worth being explicit here that we're talking about searching GitHub.com.

Reply:
I updated to make it "GitHub code search" so its a bit more explicit, but hesitate to put GitHub.com because it could be mistaken for using the code search UI on GitHub.com to find and add repositories, whereas the main advantage of this feature is being able to use search directly in the extension. "Using GitHub.com code search in the extension to add repositories to a custom list" is probably most accurate but quite long